dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -5,27 +5,94 @@
 using OpenQA.Selenium.Edge;
 using OpenQA.Selenium.Firefox;
 using OpenQA.Selenium.Safari;
+
 // ReSharper disable InconsistentNaming
 
 namespace Simple.Sauce
 {
     public class SauceOptions
     {
         private const string DEFAULT_BROWSER_VERSION = "latest";
+
         private const string DEFAULT_PLATFORM = "Windows 10";
 
+        //TODO not fond of this name
+        public static List<string> W3CAllowedOptionsList = new List<string>(new[]
+            {
+                "BrowserName",
+                "BrowserVersion",
+                "PlatformName",
+                "PageLoadStrategy",
+                "AcceptInsecureCerts",
+                "Proxy",
+                "SetWindowRect",
+                "Timeouts",
+                "StrictFileInteractability",
+                "UnhandledPromptBehavior"
+            }
+        );
+
+        private static readonly List<string> _sauceAllowedOptions = new List<string>(new[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_sauceAllowedOptions'.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)